### ISSUE TITLE
Parametrized training templates

### ISSUE THREAD

**Issue by juansensio** [earthpulse/eotdl#154]
Create parametrized training templates.

Requires generic Q1+ wrappers -> This is a class in PytorchEO called "EOTDLQ1Dataset" that accepts the name of the dataset as a parameter.

**Comment by juansensio** [comment]
Use cases:

- User goes to their workspace, select the template, change the code and execute.
- User click on train -> redirect to trianing UI -> input parameters -> execute. Parameters: dataset, resources (cpu/gpu), global hyperparameters (number of epochs, learning rate, batch size, ... but not really much else).

**Comment by juansensio** [comment]
@juansensio provide two notebook examples with different datasets but for the same task to identify where the parameters should go.

**Comment by achtsnits** [comment]
List view should show invocations separately per dataset and per user!

**Comment by juansensio** [comment]
A first iteration of the generic wrapper can be found here: https://github.com/earthpulse/pytorchEO/blob/main/examples/eotdl_wrapper.ipynb

It works for classification tasks and Sentinel 2 for the moment, we will improve as new Q1 datasets are ingested and the metadata specification is improved. 

Use this to implement a first parametrized training template, using the EuroSAT-Q1-small as demo.

**Comment by juansensio** [comment]
Resuming this task @achtsnits @Schpidi, what is the status on this ?

**Comment by juansensio** [comment]
Thanks, I will add these instructions in the docs as well.

**Comment by earthpulse** [comment]
Tried, and worked fine. We can close this issue and open new ones for particular models/datasets we want to support.
